### ISSUE TITLE
Promoting K.TV More Widely

### ISSUE THREAD

**Issue by JoshuaKimsey** [riquezjp/kitchenTV#10]
So with K.TV being fairly well developed at this point, although the issue of combining your version and mine still exist, we should start trying to promote K.TV more widely so people will discover it. I was thinking it might be a good idea to make a thread on the official RasPi forums about, as that's definitely one of the main hubs for stuff about the RasPi. Also, maybe a new YouTube video is in order to display the work we did on our versions? Would serve as a good sequel to your original video! 

Let me know what you think! Would love to hear other ideas you might have! 😁

**Comment by riquezjp** [comment]
Hi J95. Yes these are both great ideas. 
If I have time I will try & do another video, the original is not very exciting. Something short & snappy would be good.

**Comment by JoshuaKimsey** [comment]
Indeed! I might be able to help with this endeavor! I have to ask, which version should be shown off though? You're more compatible version, or my feature rich version?
